tests/test_cli.py

```diff
@@ -67,7 +67,7 @@ def test_no_config(self):
 
     def test_config_compatibility(self):
         for config in sorted(self.test_config_path.glob("**/*.yaml")):
-            if "invalid" in str(config):
+            if "invalid" in str(config) or "mpi" in str(config):
```

Comment:
This should not be skipped, it is expected that users can have old configs that do not have `mpi` and code will still run (just we will not use `mpi`). If this test fails right now without it we have issues and need to fix the defaults.

Reply:
This is skipping testing of test configs that have "mpi" in the file name. The problem that I ran into with this was the GitHub actions run saying that `mpirun` is not installed.

Reply:
Yes. You need to make a test skipper similar to require_GPU in test_utils/testing.py to decorate it with.

Reply:
Makes sense.